lib/src/configuration.cc

```diff
@@ -156,6 +161,11 @@ HW::ParseResult Configuration::parseOptions(int argc, char *argv[])
     // This is primarily for testing.
     master_uris_.clear();
 
+    // remember the path to the pxp-agent executable used to start
+    // this process, it is supposed to be used to start executables
+    // which are installed alongside the pxp-agent executable
+    exec_prefix_ = fs::absolute(fs::path(argv[0]).parent_path());
```

Comment:
In an actual install, we'll use the absolute path so it shouldn't matter. For my local testing, I usually run via `./bin/pxp-agent`; that seems to break when using a relative path.